support/SubmitScripts/Frontera.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #SBATCH -J MyJobName       # Job name
-#SBATCH -o MyJobName.o%j   # Name of stdout output file
-#SBATCH -e MyJobName.e%j   # Name of stderr error file
+#SBATCH -o spectre.stdout  # Name of stdout output file
+#SBATCH -e spectre.stderr   # Name of stderr error file
 #SBATCH -p small           # Queue (partition) name - for 3+ nodes use 'normal'
```

Comment:
It would be nice to have a generic python wrapper that combines command arguments and reading machine-specific files to generate a submit script and submit the job.  Command line arguments are used for job-specific parameters (number of nodes, run time, etc.), while the machine-specific files are used for machine-specific info.  This is not a change request for this PR.

Reply:
FYI, I have done some work in this direction